brightchain-lib/src/lib/interfaces/blockEncryption.ts

```
@@ -0,0 +1,20 @@
+ */
+export interface IEncryptedBlockCreator {
+  from(
+    type: BlockType,
+    dataType: BlockDataType,
+    blockSize: number,
+    data: Buffer,
+    checksum: ChecksumBuffer,
+    creator: BrightChainMember,
+    dateCreated?: Date,
+    lengthBeforeEncryption?: number,
+  ): Promise<any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)